vendor/github.com/elastic/beats/libbeat/cmd/instance/beat.go

```diff
@@ -110,6 +110,9 @@ type beatConfig struct {
 	// elastic stack 'setup' configurations
 	Dashboards *common.Config `config:"setup.dashboards"`
 	Kibana     *common.Config `config:"setup.kibana"`
+
+	// Migration config to migration from 6 to 7
+	Migration *common.Config `config:"migration.6_to_7"`
```

Comment:
does this affect us?

Reply:
It can minimally - setting `migration.6_to_7.enabled=true` triggers creation of 14 non-conflicting aliases.  We won't document this as there isn't a known reason to do this right now.